main.js

```diff
@@ -0,0 +1,99 @@
+console.log('hi');
+
+//smokescreen says surprise in console after 2 seconds (2000 ms)
+var intervalId = window.setInterval(function(){
+  console.log('surprise');
+}, 2000);
+
+//smokescreen clearInterval that stops above code
+window.clearInterval(intervalId);
+
+var minutes;
+var seconds;
+var input = document.querySelector('input');
+var taskButton = document.querySelector('.addTask');
+var startButton = document.querySelector('.start');
+var ol = document.querySelector('ol');
+var clockFace = document.querySelector('.countDown');
+var timeOutId;
+var complete = document.querySelector('.complete');
+
+//function to decrease time
+// var secondsId = setInterval(function(){
+//   console.log(seconds -= 1);
+// }, 1000);
+
+//turns text typed into input to tasks on HTML
+var handleInput = function(event) {
+  // console.log('keyCode', event.keyCode, event.target.value);
+  if (event.keyCode === 13) {
+    // console.log('pressed enter')
+    var li = document.createElement('li');
+    li.textContent = event.target.value;
+    ol.appendChild(li);
+    event.target.value = "";
+  };
+}
+
+//create time function (25 mins = 1500 seconds * 1000 milliseconds)
+var twentyFiveMins = function(mins) {
+  seconds = 60;
+  minutes = mins - 1;
+
+  var minutesId = setInterval(function(){
+  minutes -= 1;
+  clockFace.textContent = (minutes + ":" + seconds);
+  }, 60000 / 120); // divided by 120 to speed up timers
+
+  var secondsId = setInterval(function(){
+  seconds = seconds - 1;
+  clockFace.textContent = (minutes + ":" + seconds);
+  }, 1000 / 120);
+
+  secondReset = setTimeout(function(){
```

Comment:
Rather then setting two timers for minutes and seconds, think about how you could run one timer for 25 * 60 * 1000 milliseconds and then display it as minutes and seconds. This might fix the negative numbers in your seconds as well.